flexwrfoutput/openfiles.py

```diff
@@ -0,0 +1,77 @@
+"""
+Functions to handle raw netCDF output of FLEXPART-WRF.
+
+Meant to be used before the postprocessing with the accessor.
+"""
+from pathlib import Path
+from typing import Tuple, Union
+
+import xarray as xr
+
+
+def combine(flxout: xr.Dataset, header: xr.Dataset) -> xr.Dataset:
```

Comment:
```suggestion
def _combine_output_and_header(flxout: xr.Dataset, header: xr.Dataset) -> xr.Dataset:
```

Reply:
Done in f0771b080aec6617e9a921e8d0c2f366c012cfaf